code/controllers/subsystems/initialization/robots.dm

```diff
@@ -60,8 +43,8 @@ SUBSYSTEM_DEF(robots)
 		if(modules[include_override])
 			.[include_override] = modules[include_override]
 
-/datum/controller/subsystem/robots/proc/get_mmi_type_by_title(check_title)
-	. = mmi_types_by_title[lowertext(trim(check_title))] || /obj/item/device/mmi
+/datum/controller/subsystem/robots/proc/get_mmi_type_by_title(check_title) //this is a lazy ass fix but i dont know how else to do this without breaking everything
+	. = /obj/item/device/mmi
 
 /datum/controller/subsystem/robots/proc/get_mob_type_by_title(check_title)
-	. = mob_types_by_title[lowertext(trim(check_title))] || /mob/living/silicon/robot
+	. = /mob/living/silicon/robot
```

Comment:
> This is actually really bad. Did you even check what these functions do? I get not knowing something but then ask someone on the dev team, don't put up a pr where you know something is shitcode.

these procs controlled if a robot will get an mmi in them or a positronic brain by their alt title, since i got rid of the alternate titles all borgs will get an MMI in them, instead of replacing every god damn reference to this proc, i just make it refer to the mmi only

Reply:
First of all, there is a find function. Secondly, leaving a proc that only returns the same thing around because you cant be bothered to replace stuff properly is shitcode. Please replace references to this proc appropriately.

Reply:
Im gonna resolve my earlier statement about the two lists since you've explained what the proc is for, but please replace references with the appropriate code.